llvm/lib/Target/ARM/ARMFrameLowering.cpp

```diff
@@ -2128,14 +2148,17 @@ bool ARMFrameLowering::spillCalleeSavedRegisters(
   auto IsDPRCS1 = [&CheckRegArea](unsigned Reg) {
     return CheckRegArea(Reg, SpillArea::DPRCS1);
   };
+  auto IsGPRCS3 = [&CheckRegArea](unsigned Reg) {
+    return CheckRegArea(Reg, SpillArea::GPRCS3);
+  };
 
   // Windows SEH requires the floating-point registers to be pushed between the
   // two blocks of GPRs in some situations. In all other cases, they are pushed
   // below the GPRs.
   if (PushPopSplit == ARMSubtarget::SplitR11WindowsSEH) {
```

Comment:
Can this `if` be removed now that the blocks are always pushed in the same order?